kubernetes/README.md

```diff
@@ -0,0 +1,38 @@
+* Helm chart for BKVM
+
+In order to install BKVM in an existing cluster that runs BookKeeper you can use this Helm Chart.
+
+** Notes
+
+This helm chart is very simple and it only starts a pod that runs BKVM service and the embedded HerdDB database.
+The default configuration should work with simple clusters, without authentication on ZooKeeper services.
+
+Usually setting metadataServiceUri is enough if you do not have authentication for ZooKeeper.
+
+** Contributing
+
+Feel free to give your feedback using GitHub issues or send Pull Requests with the enhancements you do locally.
+
+** Installing the chart
+
+Ensure that you have KUBECONFIG properly configured and than `helm` works properly.
+
+Clone this repository and then run this command, overring properly the metadataServiceUri parameter.
+
+```
+helm install bkvm -f bkvm/values.yaml bkvm --set metadataServiceUri=zk://pulsar-zookeeper-ca:2181/ledgers
```

Comment:
It did in on purpose because actually it is an example that works.
there is not Helm Chart for BookKeeper, only for Apache Pulsar and for Pravega.io

I would add a section for Pulsar and one for Pravega.

When we will have a Helm Chart for HerdDB that would be also a good example.

There are plenty of Pulsar users who are asking for BKVM, and having the example that works out of the box is very useful in my opinion